src/civis/_get_api_key.py

```diff
@@ -0,0 +1,16 @@
+import os
+
+
+def get_api_key(api_key):
```

Comment:
This isn't new code. Moved this function here from `_utils.py` (now renamed `_retries.py` for clarity and not having a cluttered "utilities" module around).